### ISSUE TITLE
Invalid version range

### ISSUE THREAD

**Issue by asmaamohamed1989** [udacimak/udacimak#198]
**When I try to use udacimak I get an error Invalid version range  

**

**To Reproduce**
5. _(Add more or edit as appropriate)_

**Expected behavior**

**Udacimak Log**
Relevant portion of the logs from `udacimak.info.log`
and/or `udacimak.error.log` files:

```shell
INSERT_LOG_HERE
```

**Screenshots**

**Desktop (please complete the following information):**
 - Node version: (`node --version`)
 - npm version: (`npm --version`)
 - Udacimak Version: (`udacimak --version`)

**Course Information**
 - Nanodegree or Course name: [e.g. Front End Web Developer Nanodegree]
 - Version: [e.g. v1.0.0]
 - Locale: [e.g. en-us]

**Additional context**
![Capture](https://user-images.githubusercontent.com/59483276/115097903-2fd89580-9f2d-11eb-83d8-a28795499efd.PNG)

**Comment by reachayush** [comment]
![image](https://user-images.githubusercontent.com/33036718/118387963-2121f300-b63f-11eb-9a05-4844211f2181.png)

Can someone please help with this issue? I am not able to understand the problem here

**Comment by Madankumar90** [comment]
you got to replace the node module file with the @SJ029626  comment https://github.com/udacimak/udacimak/issues/207 and the drive's node module it will work. i just tried it out.

**Comment by Madankumar90** [comment]
@reachayush you can try the above method. i don't know much about the issue you have put up. Even though after getting these errors. you can navigate to the C:\Users\...\AppData\Roaming\npm and you will find a folder called node_modules. Just replace this folder with the  #207   mentioned workaround.

**Comment by reachayush** [comment]
@Madankumar90 So after that just nothing else to do...Just start using it?

**Comment by Madankumar90** [comment]
yes you can download required nanodegrees, by following required next steps, which i think you know. just use those udacimak commands in prompt and it should work.
